packages/mobile/android/app/build.gradle

```diff
@@ -147,7 +147,7 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  * on project.ext.react, JavaScript will not be compiled to Hermes Bytecode
  * and the benefits of using Hermes will therefore be sharply reduced.
  */
-def enableHermes = project.ext.react.get("enableHermes", false);
+def enableHermes = project.ext.react.get("enableHermes", true);
```

Comment:
I don't think there's a configuration issue here.

This changes the default value from `true` to `false` in case `enableHermes` is not set.
But we do set it to `true` here: https://github.com/valora-inc/wallet/blob/4dff4f4dd442d239af94313fcf364d6f814ba0ad/packages/mobile/android/app/build.gradle#L77

Also Sentry reports confirm Hermes is enabled in 1.26.0
Example: https://sentry.io/organizations/valora-inc/issues/2965439425/events/a9e65be4d8af47439174d626eb58c07d/?project=1250733

Reply:
Changed back to original in https://github.com/valora-inc/wallet/pull/2001/commits/c9a9b33392964698eedc51cac3e2fd0585d0a56f!